docs/system/container-metrics.md

```diff
@@ -0,0 +1,99 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.utilization(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.utilization` | Gauge | `1` | Recent CPU utilization for the container. [1] |
+
+**[1]:** CPU usage percentage normalized by the number of CPU cores. The value range is [0.0,1.0].
```

Comment:
Does this normalisation also include entitlements? Say I am entitled to 250ms of 1 cpu, i.e. 0.25 cpus, will that be reflected as well?

Reply:
I'm not sure if I get what this entitlement means :) . Is this for example a resource limit? If so, I would handle it as a different metric. 

This one stands for the percentage of the host's CPU the container is using similarly to what `docker stats` would provide.

Reply:
I was referring to [CPU resource units in Kubernetes](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu). Here I can define, how much "millicpus" a container can use. If I define 500m or 0.5 and the container is using 250m, will this be reported as 50% of the 500m or 250m of a full cpu (ignoring the limit). If the latter is the case, how can I transport the limit in that case, to investigate whether the container is saturating its assigned cpu?

Reply:
My proposal would the following:

- `container.cpu.utilization` will report the % utilization of a full cpu.

- If we want to report the % utilization of the limit we can use a different metric like `container.cpu.limit.utilization`

Reply:
That would be an option as well @KarstenSchnitter, thank's ! Any suggestion on how this attribute should be named?
Also I would love to hear more thoughts from @open-telemetry/semconv-container-approvers here :).

Reply:
related: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24905#issuecomment-1688903953

Reply:
@ChrsMark the attribute name could follow the [hw.logical_disk.usage](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/hardware-metrics.md#hwlogical_disk--logical-disk-metrics) which has a `state` attribute with values `used` and `free`.

Reply:
Agreed this should be utilization against system. This is how docker reports this metric too.